roles/openshift_logging_elasticsearch/templates/operator_adopt.j2

```diff
@@ -0,0 +1,71 @@
+apiVersion: "elasticsearch.redhat.com/v1alpha1"
+kind: "Elasticsearch"
+metadata:
+  name: "{{ es_cluster_name }}"
+spec:
+  elasticsearchConfig:
+    image: "{{ openshift_logging_elasticsearch_image }}"
+    resources:
+      limits:
+{% if es_cpu_limit is defined and es_cpu_limit is not none and es_cpu_limit != '' %}
+        cpu: "{{ es_cpu_limit }}"
+{% endif %}
+        memory: "{{ es_memory_limit }}"
+      requests:
+        cpu: "{{ es_cpu_request }}"
+        memory: "{{ es_memory_limit }}"
+  nodes:
+{% for node in es_cluster_nodes %}
+    - nodeRole: clientdatamaster
```

Comment:
```
roles:
- client
- data
- master
```
this way?^

Reply:
or `roles: [client, data, master]`

Reply:
I think this makes sense.

Reply:
both should resolve to be the same internally. I think it may be easier using that approach, I had to look up what the correct string was, using a list removes needing to do that for customers.

Reply:
Updated to be
```yaml
roles:
- client
- data
- master
```

Reply:
updated in the operator